### DISCUSSION TITLE
[Feature request] Native speaker fill / Multi-channel device support

### DISCUSSION THREAD

**Discussion by Hexality** [ppy/osu#26028] [Ideas] [answered]
Can we get native support for devices with multi channel configuration? some devices (like my HT, which has a 5.1 configuration) do stereo separation in a way that only the subwoofer reproduces bass, and due to that, the audio from the game becomes weird due to the lack of frequencies in the low range. If the game had something like realtek's speaker fill (which got discontinued) but native, people would be able to use devices like this without having to use third-party software worrying about the amount of latency that would be introduced by such software.

Example: AIMP Channel Expander (Exclusive to the music player)
![image](https://github.com/ppy/osu/assets/17398632/484b9303-3e5e-4d8f-a552-d6e0d7591a24)

**Comment by peppy** [comment, marked as answer]
out of scope i think

**Reply by Hexality** [reply]
nvm, migrated to linux, found out here we can do that through pipewire natively.

**Reply by Hexality** [reply]
![image](https://github.com/ppy/osu/assets/17398632/bac44eef-c8d6-46d8-9d93-7dd1c36e6cd9)
